packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -297,6 +297,11 @@ class WebFs {
 class BuildDaemonCreator {
   const BuildDaemonCreator();
 
+  // TODO(jonahwilliams): find a way to get build checks working for flutter for web.
+  static const String _ignoredLine1 = 'Warning: Interpreting this as package URI';
```

Comment:
This is coming from the dart VM. It wouldn't happen if the entrypoint wasn't under `lib/`.

We might want to file a request to allow us to silence this or something.